Sources/BuildSystemIntegration/SwiftPMBuildSystem.swift

```diff
@@ -547,8 +558,12 @@ extension SwiftPMBuildSystem: BuildSystemIntegration.BuildSystem {
     return []
   }
 
-  package func generateBuildGraph(allowFileSystemWrites: Bool) async throws {
-    try await self.reloadPackage(forceResolvedVersions: !isForIndexBuild || !allowFileSystemWrites)
+  package func generateBuildGraph() async throws {
+    try await self.reloadPackage(forceResolvedVersions: !isForIndexBuild)
```

Comment:
`forceResolvedVersions` is now always `!isForIndexBuild`, is it worth keeping it as a parameter?

Reply:
Nice catch, I didn’t notice we can remove it even one level further. 👍🏽